docs/tutorial.md

```diff
@@ -555,11 +555,14 @@ You can also see the statuses of the individual `TaskRuns`.
 
 ## Running this tutorial locally
 
-This section provides guidelines for completing this tutorial on your local workstation.
+This section provides guidelines for completing this tutorial on your local workstation on:
 
-### Prerequisites
+* [Docker for Desktop](#prerequisites-docker-for-desktop)
```

Comment:
minor nit, suggest matching bullet style with rest of document (using `-` instead of `*`)

Reply:
@sbwsg was going to fix this, but looks like the PR already merged... I can open another one that'll clean this up (and fix the resource usage too), or wait and include it alongside something more meaningful if you think this is too trivial for a standalone PR.

Reply:
Small PRs are good PRs :). Fixing in a follow-up is totally a good way to go.  Thanks @psschwei !